package.json

```diff
@@ -45,7 +45,9 @@
     "ngx-pagination": "^3.2.0",
     "rxjs": "5.4.0",
     "sw-toolbox": "3.6.0",
-    "zone.js": "0.8.12"
+    "zone.js": "0.8.12",
+    "cordova-android": "^6.2.3",
```

Comment:
Interesting that these weren't here before? Did you perhaps `npm install` them rather than `ionic cordova platform add`?

Reply:
Yes, I ran npm install

Reply:
I'm not sure if that's a problem then? At the very least it seems unnecessary since `ionic cordova platform whatever` also interacts with NPM while also keeping your `platforms/` directory maintained, right?

Reply:
@werebus, right. So maybe I should remove these then. Sorry I didn't see this comment before I merged 🐑.